src/main/webapp/app/exercises/programming/manage/update/programming-exercise-update.component.ts

```diff
@@ -261,14 +276,39 @@ export class ProgrammingExerciseUpdateComponent implements OnInit {
         return this.selectedProjectTypeValue;
     }
 
+    onTestRepositoryProjectTypeChange(type: JavaTestRepositoryProjectType) {
+        this.selectedTestRepositoryProjectType = type;
+        return type;
+    }
+
+    /**
+     * Updates the test repository project type. This only applies to Java exercises with project type 'Plain Java'
+     * @param type the type to update to
+     */
+    set selectedTestRepositoryProjectType(type: JavaTestRepositoryProjectType) {
```

Comment:
I don't think this makes sense as you currently only display the maven option if a maven option is selected above.

Reply:
Removed in a6da54c.